src/stylesheets/core/_font-definitions.scss

```diff
@@ -0,0 +1,146 @@
+// -----------------------------------------------------
+
+// FONTS
+
+$font-stack-system: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+
+$uswds-font-definitions: (
+  system: (
+    name: null,
+    stack: $font-stack-system,
+    system-font: true,
+    cap-height: 362px,
+  ),
+  georgia: (
+    name: 'Georgia',
+    stack: $georgia,
+    system-font: true,
+    cap-height: 346px,
+  ),
+  helvetica: (
+    name: 'Helvetica',
+    stack: $helvetica,
+    system-font: true,
+    cap-height: 357px,
+  ),
+  merriweather: (
+    name: 'Merriweather Web',
+    dir: 'merriweather',
+    stack: $georgia,
+    system-font: false,
+    cap-height: 372px,
+    variable-font: false,
+    roman: (
+      100: false,
+      200: false,
+      300: 'merriweather-light-webfont',
+      400: 'merriweather-regular-webfont',
+      500: false,
+      600: false,
+      700: 'merriweather-bold-webfont',
+      800: false,
+      900: false,
+    ),
+    italic: (
+      100: false,
+      200: false,
+      300: false,
+      400: 'merriweather-italic-webfont',
+      500: false,
+      600: false,
+      700: false,
+      800: false,
+      900: false,
+    ),
+  ),
+  roboto-mono: (
```

Comment:
I know we explicitly recommend using Source Sans Pro and Merriweather now - how do Roboto Mono and United Sans work into that recommendation? Are these just to extend the number of available options or are there specific instances when these new font families should be chosen? (Sorry if this has already been covered elsewhere).

Reply:
Hasn't been covered yet, but I consider this a topic for upcoming guidance. But also even in the absence of guidance, by starting to build a library of font definitions, we can begin to use the system's new ability to normalize font sizes — and be more simpler and more programmatic about building our `@font-face` rules